tests/functional/firefox/test_welcome_page1.py

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.welcome.page1 import FirefoxWelcomePage1
+
+
+@pytest.mark.sanity
```

Comment:
Sanity tests are reserved for IE9, so I don't think this marker needs to be here.

Reply:
🍝